tools/runfiles/runfiles.rs

```diff
@@ -97,29 +99,34 @@ fn find_runfiles_dir() -> io::Result<PathBuf> {
             break;
         }
         // Follow symlinks and keep looking.
-        binary_path = binary_path.read_link()?;
-        if binary_path.is_relative() {
-            binary_path = env::current_dir()?.join(binary_path)
+        let link_target = binary_path.read_link()?;
+        binary_path = if link_target.is_absolute() {
+            link_target
+        } else {
+            let link_dir = binary_path.parent().unwrap();
+            env::current_dir()?.join(link_dir).join(link_target)
```

Comment:
This was missing the `link_dir` component before.

Reply:
Add this to PR description?